toolbox/githubctl/main.go

```diff
@@ -125,6 +125,13 @@ func TagIstioDepsForRelease() error {
 		if !exists {
 			return fmt.Errorf("ill-defined %s: unable to find %s", istioVersionFile, dep.Name)
 		}
+		if tagExists, err := githubClnt.ExistTag(dep.RepoName, releaseTag); err != nil {
+			log.Printf("Tailed to check if tag [%s] exists on repo %s\n", releaseTag, dep.RepoName)
+			return err
+		} else if tagExists {
```

Comment:
don't you want to check if reference the tag points to is the same as the new one ? if is the same, continue, otherwise fail